### ISSUE TITLE
KDE systemsettings: Network Settings ->SSL Preferences unavailable

### ISSUE THREAD

**Issue by adam-burns** [dyne/dynebolic#22]
Under dyneIV-20240116.iso image, Settings application page reports error:

```
Could not load plugin from kcm_ssl: The shared library was not found.

Possible reasons:
- An error occurred during your last system upgrade, leaving an orphaned control module behind
- You have old third party modules lying around.
check these points carefully and try to remove the module mentioned in the error message. If this fails, consider contacting your distributor or packager.
```

On (working) vanilla Devuan Daedalus system:
```
$ dpkg -S kcm_ssl.so
libkf5kdelibs4support5-bin: /usr/lib/x86_64-linux-gnu/qt5/plugins/kcm_ssl.so
```
On dynebolic image:
```
$ dpkg -S kcm_ssl.so
dpkg_-query: no path found matching pattern *kcm_ssl.so*
```
Note that
`$ apt install libkf5kdelibs4support5-bin
`
fixes the systemsettings page, but I'm not yet sure that just installing this package is sufficient to ensure working ssl preference changes (ie there may be a better 'root cause' kde package that pulls in this package as dependency).

**Comment by adam-burns** [comment]
Currently in [desktop-kde](https://github.com/dyne/dynebolic/blob/master/dyneIV/packages/desktop-kde), the minimal `kde-plasma-desktop` meta package is installed.

After testing, neither `apt install kde-standard` nor `apt install kde-full` bring in `libkf5kdelibs4support5-bin` to enable this preference menu. Various  `apt-cache rdepends ...` outputs also gave no hints as to a minimal approach.

This hints to me that either this preference pane is purposely not active nor needed until other KDE applications or subsystems are installed, or it is an upstream meta-packaging bug.

Thus, in the interests of keeping dynebolic a clean minimal install image, I'm closing this issue without action until it's proven to have an impact.

**Comment by Sakrecoer** [comment]
@adam-burns 
Thank you very much! :purple_heart: 

I believe this issue should remain open: it's definitely something that needs to be addressed. There will be more issues and the settings pane is big enough that we risk forgetting this one.

**Comment by adam-burns** [comment]
> @adam-burns Thank you very much! 💜
> 
> I believe this issue should remain open: it's definitely something that needs to be addressed. There will be more issues and the settings pane is big enough that we risk forgetting this one.

Feel free to re-open, however note the solution [above](https://github.com/dyne/dynebolic/issues/22#issue-2092791133) fixes the preference panel layout but is possible to not to function as the user expects. You could try the install and test the functionality - if it works you or I can push the change. My preference is to keep it as it is until functionality of a fix is verified.

**Comment by adam-burns** [comment]
After gathering thoughts and quick testing results with @Sakrecoer & @jaromil , the proposed fix above was pushed in b13a2694060d9fa131daea74294e6549f22acc9a
